### PR TITLE
Release: 2.16.0

### PR DESCRIPTION
**Pull request by taylorjdawson**

### CODE REVIEW

File: packages/demo/webpack.config.js [hunk 1]
```diff
@@ -8,6 +8,9 @@ const prod = mode === 'production'
 module.exports = {
   entry: './src/main.js',
   resolve: {
+    fallback: {
```
Comment (Adamj1232) [inline packages/demo/webpack.config.js:11]: @taylorjdawson Can we add this to the webpack config in the Core Readme?

File: packages/core/src/utils.ts [hunk 2]
```diff
@@ -12,22 +12,23 @@ import type {
...
+  polygonIcon,
+  binanceIcon,
+  questionIcon,
+  fantomIcon,
+  optimismIcon,
+  celoIcon,
+  avalancheIcon,
+  harmonyOneIcon,
+  arbitrumIcon
+} from './icons'
```
Comment (aaronbarnardsound) [inline packages/core/src/utils.ts:31]: Does this need to be `'./icons/index.js'` to maintain ESM compatibility?

File: packages/core/src/views/account-center/Maximized.svelte [hunk 3]
```diff
@@ -10,10 +10,8 @@
   import connect from '../../connect.js'
   import disconnect from '../../disconnect.js'
   import { state } from '../../store/index.js'
-  import WalletAppBadge from '../shared/WalletAppBadge.svelte'
   import { getDefaultChainStyles, unrecognizedChainStyle } from '../../utils.js'
-  import SuccessStatusIcon from '../shared/SuccessStatusIcon.svelte'
-  import NetworkSelector from '../shared/NetworkSelector.svelte'
+  import { NetworkSelector, SuccessStatusIcon, WalletAppBadge } from '../shared'
```
Comment (aaronbarnardsound) [inline packages/core/src/views/account-center/Maximized.svelte:14]: Same with all of these imports, I think we need the full extension to maintain compatibility with ESM standard.

File: packages/demo/webpack.config.js (hunk 1 above)
Comment (taylorjdawson) [reply packages/demo/webpack.config.js:11]: Good idea!

**Review by Adamj1232 — APPROVED** [review]
Good work man!! Fire it up 🔥 🚀  LGTM!

**Pull request merged**